### PR TITLE
KIALI-1725: Standardize package.json via sort-package-json 

### PR DESCRIPTION
**Pull request by mtho11**
** Describe the change **
Standardize the package.json structure via: - https://www.npmjs.com/package/sort-package-json

This arose out of the need to see both _dependencies_ and _dev-dependencies_ near one another -- as their versions are linked -- most especially with the '@types/*' stuff for typescript definitions.
Obviously, others have already solved this issue.

** Issue reference **

https://issues.jboss.org/browse/KIALI-1725

** Backwards compatible? **
yes

### CODE REVIEW

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#197](https://paste.fedoraproject.org/paste/wI0jadGpmkpD8bX-zTErCA)
  * :heavy_check_mark: Istio-and-Kiali-on-Openshift [#[1566]](https://paste.fedoraproject.org/paste/-Yn9F94rgmuW~oOmMkChbw)
  * :heavy_check_mark: Istio-Upstream [#[421]](https://paste.fedoraproject.org/paste/7nDJESAQEn1jeMmhPRWlRA)
  * :heavy_check_mark: Kiali-Istio-Checker [#[1688, 1687]](https://paste.fedoraproject.org/paste/syg3lDEJv-7uAZHyBCgM6A)
  * :heavy_check_mark: install-bookinfo [#[937]](https://paste.fedoraproject.org/paste/yuPV5ndxw6v8O1B3nGxxCw)
  * :heavy_check_mark: Kiali-on-Openshift [#[944]](https://paste.fedoraproject.org/paste/UJCtLp9nv4qEv-eLvDtdrg)
  * :heavy_check_mark: run-kiali-e2e-tests [#[363]](https://paste.fedoraproject.org/paste/U29vNftXvU6fKbRFk-7gPg)


KIALI_NAME=kialiuipr741-istio-system

File: package.json [hunk 1]
```diff
@@ -99,37 +111,27 @@
...
   },
+  "engines": {
+    "node": ">=8.0.0 ",
+    "npm": ">=5.0.0 ",
+    "yarn": ">=1.0.0 "
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "homepage-comment": [
```
Comment (josejulio) [inline package.json]: This was intended to go above "homepage", if not there, is not worth to have it at all.

File: package.json [hunk 2]
```diff
@@ -99,37 +111,27 @@
...
-  "homepage": "./",
-  "publishConfig": {
-    "access": "public"
-  },
-  "engines": {
-    "node": ">=8.0.0 ",
-    "npm": ">=5.0.0 ",
-    "yarn": ">=1.0.0 "
-  }
+  "script-comments": [
```
Comment (josejulio) [inline package.json]: This was intended to go above "script", if not there, is not worth to have it at all.

File: package.json (hunk 1 above)
Comment (mtho11) [reply package.json]: This is the generated order by the sort-package-json script.
I'm fine moving the comments to a more logical place though.

File: package.json (hunk 2 above)
Comment (mtho11) [reply package.json]: @josejulio please retest, I made your changes.

**Comment by rhqci** [comment]
:heavy_check_mark: Jenkins CI: kiali-ui-pr-e2e-test [#202](https://paste.fedoraproject.org/paste/umbd5fqxkq732UMnIghCtA)
  * :red_circle: Istio-and-Kiali-on-Openshift [#[1578]](https://paste.fedoraproject.org/paste/7hSTgqejMgFdr1YoywjaUA)
  * :heavy_check_mark: Istio-Upstream [#[432]](https://paste.fedoraproject.org/paste/tOfQWixgO0ynxwasnCGLZg)
  * :heavy_check_mark: Kiali-Istio-Checker [#[1718, 1719]](https://paste.fedoraproject.org/paste/2gnnzUIotyU9obJ17StBPw)
  * :heavy_check_mark: install-bookinfo [#[950]](https://paste.fedoraproject.org/paste/Jk5V3UtmjtlitrC7-pVKwA)
  * :heavy_check_mark: Kiali-on-Openshift [#[964]](https://paste.fedoraproject.org/paste/jCYCGxZmMa0ziH-1IuQCMg)
  * :red_circle: run-kiali-e2e-tests [#[376]](https://paste.fedoraproject.org/paste/TIQPOvbSe~gsSZAJYv9H9g)


KIALI_NAME=kialiuipr741-istio-system

**Pull request merged**